packages/ui/src/assets/css/primereact-override/icon.css

```diff
@@ -0,0 +1,7 @@
+:root {
+  --icon-primary-color: var(--dz-primary-color);
+}
+
+.pi-cog {
+  color: var(--icon-primary-color) !important;
+}
```

Comment:
we cannot define color for a icon class here. We may want to use the icon in many places with different color. This will cut out that possibility. I think you want to apply this css in table's actions. We can leave that for now, we will later update it with our own primary button which will fix the color.